ui/app/models/kmip/config.js

```diff
@@ -3,8 +3,11 @@ import { computed } from '@ember/object';
 import { combineFieldGroups } from 'vault/utils/openapi-to-attrs';
 import fieldToAttrs from 'vault/utils/field-to-attrs';
 
+const { belongsTo } = DS;
+
 export default DS.Model.extend({
   useOpenAPI: true,
+  ca: belongsTo('kmip/ca', { async: false }),
```

Comment:
if this is a one-to-one relationship, does the `ca` model also need the `belongsTo('kmip/config', { async: false })`?

Reply:
Oops yep - I wonder if there are other relationships in our app where I've forgotten to do that 🤔. I'll fix that up